ansible/roles/fanout/templates/sonic_deploy.j2

```diff
@@ -1,56 +1,329 @@
 {
+    "DEVICE_METADATA": {
```

Comment:
it is hard to keep up the sonic configuration? why not use l2switch.j2 template embeded in the image?